lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -197,11 +197,13 @@ export class LighthouseReportViewer {
   _replaceReportHtml(json) {
     // Allow users to view the runnerResult
     if ('lhr' in json) {
-      json = /** @type {LH.RunnerResult} */ (json).lhr;
+      const runnerResult = /** @type {LH.RunnerResult} */ (/** @type {unknown} */ (json));
```

Comment:
Perfectly reasonable complaint that the existing casts are garbage. This maintains the garbage but makes it clear we know what it is :)